integration-tests/src/test/java/com/transferwise/tasks/health/ClusterWideTasksStateMonitorIntTest.java

```diff
@@ -50,6 +51,9 @@ void metricsAreCorrectlyRegistered() {
     await().atMost(30, TimeUnit.SECONDS).until(() -> tasksInErrorCountGauge.value() == 0);
     await().until(() -> stuckTasksGauge.value() == 0);
 
+    assertThat(meterRegistry.get("twTasks.state.approximateTasks").gauge().value()).isGreaterThan(-1);
+    assertThat(meterRegistry.get("twTasks.state.approximateTasks").gauge().value()).isGreaterThan(-1);
```

Comment:
I suppose we want don't want to check the same metric twice, but rather check `approximateUniqueKeys`?